packages/koa-shopify-auth/src/auth/redirection-page.ts

```diff
@@ -1,18 +1,21 @@
-export default function redirectionScript({origin, redirectTo}) {
+export default function redirectionScript({origin, redirectTo, apiKey}) {
   return `
-    <script type="text/javascript">
+    <script src="https://unpkg.com/@shopify/app-bridge"></script> <script type="text/javascript">
       document.addEventListener('DOMContentLoaded', function() {
         if (window.top === window.self) {
           // If the current window is the 'parent', change the URL by setting location.href
           window.location.href = '${redirectTo}';
         } else {
           // If the current window is the 'child', change the parent's URL with postMessage
-          data = JSON.stringify({
-            message: 'Shopify.API.remoteRedirect',
-            data: { location: '${redirectTo}' }
+          var AppBridge = window['app-bridge'];
```

Comment:
Looks great, is there any tests for this util?

Reply:
Nothing for this particular script, all this file does is render a route with this script to redirect the user out of the iframe in Safari so we can set a cookie.